cmd/clusterctl/examples/aws/generate-yaml.sh

```diff
@@ -2,30 +2,27 @@
 set -o errexit
 set -o nounset
 
+# Directories.
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-
 OUTPUT_DIR=${DIR}/out
 
-# provider-components
-export CLUSTER_CONTROLLER_IMAGE="${CLUSTER_CONTROLLER_IMAGE:-gcr.io/k8s-cluster-api/aws-cluster-controller:0.0.1}"
-export MACHINE_CONTROLLER_IMAGE="${MACHINE_CONTROLLER_IMAGE:-gcr.io/k8s-cluster-api/aws-machine-controller:0.0.1}"
+# Manager image.
+export MANAGER_IMAGE="${MANAGER_IMAGE:-gcr.io/k8s-cluster-api/aws-cluster-controller:0.0.1}"
 
-# aws credentials
-# TODO generate a secret instead of embedding directly in the YAML
-export AWS_REGION="${AWS_REGION:-us-west-2}"
-export AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}"
-export AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}"
+# Machine settings.
+export CONTROL_PLANE_MACHINE_TYPE="${CONTROL_PLANE_MACHINE_TYPE:-t2.medium}"
```

Comment:
Suggest naming this variable `MASTER_NODE_TYPE`  and `WORKER_NODE_TYPE`.

Reply:
We're actively working on avoiding new uses of `master` and trying to eliminate existing uses of `master` as we can.